### PR TITLE
feat: sets up a dev branch and a deployment branch in the same project for neonctl create-app

### PR DESCRIPTION
**Pull request by davidgomes**
CC @amitdahan @devjv 

This PR makes `neonctl create-app` set up 2 branches for each app:
* Local dev branch
* Deployment branch

### CODE REVIEW

File: src/commands/bootstrap/index.ts [hunk 1]
```diff
@@ -512,89 +876,111 @@ const bootstrap = async (props: CommonProps) => {
       throw new Error(`Creating a Next.js project failed: ${String(error)}.`);
     }
 
-    if (finalOptions.auth === 'auth.js') {
+    if (options.auth === 'auth.js') {
       // Generate AUTH_SECRET using openssl
-      const authSecret = execSync('openssl rand -base64 33').toString().trim();
+      const devAuthSecret = execSync('openssl rand -base64 33')
```
Comment (duskpoet) [inline src/commands/bootstrap/index.ts:881]: this would likely not work on windows, right?

File: src/commands/bootstrap/index.ts [hunk 2]
```diff
@@ -105,6 +115,315 @@ function writeEnvFile({
...
+  appName: string;
+  connectionString?: string;
+}) {
+  try {
+    let envVar = '';
+    if (connectionString) {
+      envVar = `DATABASE_URL=${connectionString}`;
+    }
+
+    execSync(`${envVar} ${options.packageManager} run db:migrate`, {
```
Comment (duskpoet) [inline src/commands/bootstrap/index.ts:191]: you can pass env var directly as an option

Comment (duskpoet) [reply src/commands/bootstrap/index.ts:191]: otherwise connection string with a quote would probably break the command

**Review by duskpoet — APPROVED** [review]

File: src/commands/bootstrap/index.ts (hunk 1 above)
Comment (davidgomes) [reply src/commands/bootstrap/index.ts:881]: Good call. I've just checked how the Auth.js CLI does it (https://github.com/nextauthjs/cli/blob/main/commands/secret.js#L9) and copied.

I also thought about doing a `npx auth secret` but I think that's too complex.

File: src/commands/bootstrap/index.ts (hunk 2 above)
Comment (davidgomes) [reply src/commands/bootstrap/index.ts:191]: Ah, great idea! I'm just passing it in `env` now.

**Pull request merged**